IncrementSlider/IncrementSlider.js

```diff
@@ -405,18 +407,38 @@ const IncrementSliderBase = kind({
 	},
 
 	computed: {
-		className: ({orientation, styler}) => styler.append(orientation)
+		className: ({orientation, styler}) => styler.append(orientation),
+		decrementDisabled: ({disabled, min, value = min}) => disabled || value <= min,
+		incrementDisabled: ({disabled, max, min, value = min}) => disabled || value >= max,
+		decrementAriaLabel: ({'aria-valuetext': valueText, decrementAriaLabel, min, value = min}) => {
+			if (decrementAriaLabel == null) {
+				decrementAriaLabel = $L('press ok button to decrease the value');
```

Comment:
Can we change this string to 'press button to crease the value' or do we have any better string? Agate is touch based Auto target

Reply:
I checked other strings, but couldn't find a suitable string.
I agree to exclude "OK" from the code above.